src/current/_includes/cockroachcloud/create-cluster-api-serverless.md

```diff
@@ -0,0 +1,92 @@
+**API request**
+
+To create a **Serverless** cluster, send a `POST` request to the `/v1/clusters` endpoint, specifying the following parameters:
+
+- `name`: your cluster's name, a short string with no whitespace.
+- `provider`: "GCP" or "AWS".
+- `primary_region`: (optional) Specify which region should be made the primary region. This is only applicable to multi-region Serverless clusters. This field is required if you create the cluster in more than one region.
+- `regions`: An array of strings specifying the cloud provider's zone codes. For example, for Oregon, set region_name to "us-west2" for GCP and "us-west-2" for AWS.
+- `usage_limits`:
+  - `request_unit_limit`: (int64) the maximum number of request units that the cluster can consume during the month. If this limit is exceeded, then the cluster is disabled until the limit is increased, or until the beginning of the next month when more free request units are granted. It is an error for this to be zero.
+  - `storage_mib_limit`: (int64) the maximum number of Mebibytes of storage that the cluster can have at any time during the month. If this limit is exceeded, then the cluster is throttled; only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this to be zero.
+- `spend_limit`: (integer) the maximum monthly charge for a cluster, in US cents. We recommend using `usage_limits` instead, since `spend_limit` will be deprecated in the future.
```

Comment:
```suggestion
- `name`: Your cluster's name, a short string with no whitespace.
- `provider`: "GCP" or "AWS".
- `primary_region`: (optional) Specify which region should be made the primary region. This is only applicable to multi-region Serverless clusters. This field is required if you create the cluster in more than one region.
- `regions`: An array of strings specifying the cloud provider's zone codes. For example, for Oregon, set region_name to "us-west2" for GCP and "us-west-2" for AWS.
- `usage_limits`:
  - `request_unit_limit`: (int64) The maximum number of request units that the cluster can consume during the month. If this limit is exceeded, then the cluster is disabled until the limit is increased, or until the beginning of the next month when more free request units are granted. It is an error for this to be zero.
  - `storage_mib_limit`: (int64) The maximum number of Mebibytes of storage that the cluster can have at any time during the month. If this limit is exceeded, then the cluster is throttled; only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this to be zero.
- `spend_limit`: (integer) The maximum monthly charge for a cluster, in US cents. We recommend using `usage_limits` instead, since `spend_limit` will be deprecated in the future.
```

Reply:
Is this "deprecated in the future", or is currently deprecated and will be removed in the future?